Lib/unittest/mock.py

```diff
@@ -1553,9 +1531,9 @@ def __exit__(self, *exc_info):
         del self.temp_original
         del self.is_local
         del self.target
-        for patcher in reversed(self.additional_patchers):
-            if _is_started(patcher):
-                patcher.__exit__(*exc_info)
+        exit_stack = self._exit_stack
+        del self._exit_stack
+        return exit_stack.__exit__(*sys.exc_info())
```

Comment:
Should this be passing `exc_info` parameter passed to `__exit__`? Is there a case where `exc_info` can be different from `sys.exc_info`?

Reply:
Good catch! Thank you.